QuizzApp/Core/Custom+UI/CompletionAlertViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  CompletionAlert.swift
+//  QuizzApp
+//
+//  Created by bakhva  on 19.06.23.
+//
+
+import UIKit
+
+protocol CompletionAlertDelegate: AnyObject {
+    func pressedCloseOnCompletion()
+}
+
+final class CompletionAlertViewController: UIViewController {
+    
+    weak var delegate: CompletionAlertDelegate?
+    
+    // MARK: Components
+    private let alertView: UIView = {
+        let view = UIView()
+        view.backgroundColor = QuizzAppColors.buttonColor
+        view.layer.cornerRadius = Constants.alertViewRadius
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private let alertImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = QuizzAppImages.confetti
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        
+        return imageView
+    }()
+    
+    private let congratulationsLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constants.congratulationsLabelText
+        label.font = .boldSystemFont(ofSize: Constants.congratulationsLabeFont)
+        label.textColor = .systemBackground
+        label.textAlignment = .center
+        
+        return label
+    }()
+    
+    private let pointsLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constants.pointsLabelText
+        label.font = .boldSystemFont(ofSize: Constants.pointsLabelFont)
+        label.textAlignment = .center
+        label.textColor = .link
+        
+        return label
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.addArrangedSubview(congratulationsLabel)
+        stackView.addArrangedSubview(pointsLabel)
+        
+        return stackView
+    }()
+    
+    private let dividerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .systemBackground
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private lazy var closeButton: UIButton = {
+        let button = UIButton()
+        var configuration = UIButton.Configuration.filled()
+        configuration.title = Constants.closeButtonTitle
+        configuration.baseBackgroundColor = .clear
+        button.configuration = configuration
+        button.addTarget(self, action: #selector(close), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        
+        return button
+    }()
+    
+    // MARK: View Did Load
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
```

Comment:
სახელები სრულად დაწერე, შემოკლებები არაა მიღებული
```suggestion
        configureUI()
```